prodtests/full-system-test/run-workflow-on-inputlist.sh

```diff
@@ -30,6 +30,8 @@ LOG_PREFIX="log_$(date +%Y%m%d-%H%M%S)_"
 [[ -z $OVERRIDE_SESSION ]] && export OVERRIDE_SESSION=default_$$_$RANDOM
 [[ -z $INRAWCHANNAME ]] && export INRAWCHANNAME=tf-builder-$$-$RANDOM
 
+SESSION_ID=`fairmq-shmmonitor --get-shmid --session`
```

Comment:
But this command lacks the actual session name?

Reply:
You are right, and it is even wrong. Testing did not spot it. I will fix it.

Reply:
Should be fixed now.